Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/MenuItems/HamburgerMenuItem.cs

```diff
@@ -31,11 +32,6 @@ public abstract class HamburgerMenuItem : DependencyObject
         public static readonly DependencyProperty TargetPageTypeProperty = DependencyProperty.Register(nameof(TargetPageType), typeof(Type), typeof(HamburgerMenuItem), new PropertyMetadata(null));
 
         /// <summary>
-        /// Identifies the <see cref="Tag"/> dependency property.
-        /// </summary>
-        public static readonly DependencyProperty TagProperty = DependencyProperty.Register(nameof(Tag), typeof(object), typeof(HamburgerMenuItem), new PropertyMetadata(null));
```

Comment:
Not a breaking change. Class now inherits from Control which already has a Tag property